### PR TITLE
CODEOWNERS: Committers manage commit access

### PR DESCRIPTION
**Pull request by xmulligan**
```release-note
Update CODEOWNERS for commit access
```

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -282,7 +282,7 @@ Makefile* @cilium/build
 /Documentation/cmdref/
 /Documentation/community/community.rst @cilium/contributing
 /Documentation/community/governance/ @cilium/contributing
-/Documentation/community/governance/commit_access.rst @cilium/contributing @cilium/docs-structure
+/Documentation/community/governance/commit_access.rst @cilium/tophat
```
Comment (pchaigno) [inline CODEOWNERS]: TopHat isn't committers. Committers is `@cilium/committers`.

Comment (xmulligan) [reply CODEOWNERS]: Sorry didn't see that one

Comment (xmulligan) [inline CODEOWNERS]:
```suggestion
/Documentation/community/governance/commit_access.rst @cilium/committers
```

File: CODEOWNERS [hunk 2]
```diff
@@ -358,7 +358,7 @@ Makefile* @cilium/build
 /install/kubernetes/cilium/templates/hubble* @cilium/sig-k8s @cilium/helm @cilium/sig-hubble
 jenkinsfiles @cilium/ci-structure
 /LICENSE @cilium/tophat
-/MAINTAINERS.md @cilium/contributing
+/MAINTAINERS.md @cilium/tophat
```
Comment (xmulligan) [inline CODEOWNERS]:
```suggestion
/MAINTAINERS.md @cilium/committers
```

**Review by bimmlerd — APPROVED** [review]
lgtm

**Review by aanm — APPROVED** [review]

**Comment by aanm** [comment]
@xmulligan the CI is failing https://github.com/cilium/cilium/actions/runs/5044298132/jobs/9047164429?pr=25578

**Comment by xmulligan** [comment]
Ahh sorry, from Paul's comments, I thought there was already a committers team that I wasn't aware of.

Should we make a new codeowners team for the committers (that also means we would have another place to keep up to date) or is there another solution?

**Comment by xmulligan** [comment]
I've documented it, but the job is still failing. Did I do something wrong?

**Comment by pchaigno** [comment]
> I've documented it, but the job is still failing. Did I do something wrong?

As far as I can see, the job is passing: https://github.com/cilium/cilium/actions/runs/5153954568/jobs/9281819681?pr=25578.

**Review by pchaigno — APPROVED** [review]

**Comment by bimmlerd** [comment]
Let's not forget about this. I don't think this needs CI; setting ready-to-merge, for the triager to judge.

**Comment by borkmann** [comment]
@xmulligan travis fails because of build failure related to the PR:

```
diff --git a/Documentation/codeowners.rst b/Documentation/codeowners.rst
index d335ebf4..f4f7f93d 100644
--- a/Documentation/codeowners.rst
+++ b/Documentation/codeowners.rst
@@ -40,6 +40,9 @@ repository in the Cilium project:
   Provide guidance around the best use of Cilium project continuous
   integration and testing infrastructure, including GitHub actions, VM
   helpers, testing frameworks, etc.
+- `@cilium/committers <https://github.com/orgs/cilium/teams/committers>`__
+  Committers have commit access to the project and govern who else can
+  become a committer to the project.
 - `@cilium/contributing <https://github.com/orgs/cilium/teams/contributing>`__:
   Encourage practices that ensure an inclusive contributor community. Review
   tooling and scripts used by contributors.
HINT: to fix this, run 'make -C Documentation update-codeowners'
make[1]: *** [Makefile:74: check] Error 1
make: *** [Makefile:652: postcheck] Error 2
make: *** Waiting for unfinished jobs....
  CHECK   contrib/scripts/rand-check.sh
The command "./.travis/build.sh" exited with 2.
```

**Pull request closed without merging**